qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg

```diff
@@ -254,7 +254,7 @@
                 - by_msi:
                     required_virtio_win = [1.9.35.0, )
                     uninstall_method = msi
-                    run_uninstall_cmd = 'WIN_UTILS:\AutoIt3_%PROCESSOR_ARCHITECTURE%.exe C:\${uninstall_msi_script}'
+                    run_uninstall_msi_cmd = 'WIN_UTILS:\AutoIt3_%PROCESSOR_ARCHITECTURE%.exe C:\${uninstall_msi_script}'
```

Comment:
Hi @xiagao 

Would you please share why we need rename it?

Reply:
To distinct it with the installer cmd .
run_uninstall_cmd = 'WIN_UTILS:\AutoIt3_%PROCESSOR_ARCHITECTURE%.exe C:\uninstall.au3'

Reply:
`run_uninstall_cmd` will equal `'WIN_UTILS:\AutoIt3_%PROCESSOR_ARCHITECTURE%.exe C:\${uninstall_msi_script}' `under `by_msi` condition, `uninstall_msi_script's` value is `uninstall_via_msi_x86.au3` or `uninstall_via_msi_x64.au3`.

I do not understand why `run_uninstall_cmd` will still equal to  `'WIN_UTILS:\AutoIt3_%PROCESSOR_ARCHITECTURE%.exe C:\uninstall.au3'`, because we already redefine it when the test case run into by_msi.

Reply:
The point of this patch is in .py file,   seperate uninstall_from_msi function and uninstall_from_installer in function.
And for the script name change, it's not important, just it will be more clear.

Reply:
And before the name is the same, I know that they consume the same function.
Currently, they consume different function. So it could be different, and different name is more clear when debugging issues.

Reply:
> 'WIN_UTILS:\AutoIt3_%PROCESSOR_ARCHITECTURE%.exe C:\${uninstall_msi_script}
Because when running test case of by_msi, the run_uninstall_cmd is uninstall_msi_script, which will fail on     session = run_installer_with_interaction(vm, session, test, params, run_uninstall_cmd), so I change it 

from
    session = run_installer_with_interaction(vm, session, test, params,
                                             run_uninstall_cmd)
to 
        run_uninstall_msi_cmd = utils_misc.set_winutils_letter(
            session, params["run_uninstall_msi_cmd"]
        )
        session.cmd(run_uninstall_msi_cmd)
        time.sleep(30)

Reply:
Yeah, I know, my point is maybe we don't need to change the name.
I have no more comments, waiting for others to comment, thank you.

Reply:
Ok, thanks.